demo/external-logging-elasticsearch/packager-config.yml

```diff
@@ -31,46 +31,79 @@ plugins:
     artifactId: "durable-task"
     source:
       git: https://github.com/jglick/durable-task-plugin.git
-      branch: watch-JENKINS-38381
+      # https://github.com/jenkinsci/durable-task-plugin/pull/62
+      # branch: watch-plus-UTF-8-JENKINS-31096
+      commit: e219d474cdd656ae9296314ada48605d3c87ea1c
```

Comment:
Why not use incremental versions? They are available now, for everything except `logstash` (since that is not yet merged with `master`).

Reply:
I am not sure for how long we are going to retain the Incrementals versions.
There were some plans about retention policies.

Though it's unlikely worse than depending on a branch in the fork :)

Reply:
You can expect the artifacts to be retained as long as this demo needs them. If and when there are newer MRP releases, we can switch to those.